packages/react-examples/src/react-components/Migrations/utils.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { Source } from '@storybook/addon-docs/blocks';
+import { makeStyles } from '@fluentui/react-make-styles';
+
+const codeComparisonUseStyles = makeStyles({
+  root: {
+    display: 'flex',
+    justifyContent: 'space-between',
+
+    '> *': {
+      width: '100%',
+    },
+
+    '> :not(:last-child)': {
+      marginRight: '5%',
+    },
+  },
+});
+
+export const CodeComparison = (props: { children: React.ReactElement[] }) => {
+  const { children } = props;
+  const classes = codeComparisonUseStyles();
+
+  return <div className={classes.root}>{children}</div>;
+};
+
+export const CodeLanguages: { [key: string]: string } = {
+  html: 'HTML',
+  css: 'CSS',
+  js: 'JavaScript',
+  jsx: 'React',
+};
+export const CodeExample = (props: { title?: string; children: React.ReactElement }) => {
+  const { title, children } = props;
+  // Access the raw values from the markdown source code block
+  const rawValue: string = children?.props?.children?.props?.children;
+  const isMarkdownCodeBlock = rawValue !== undefined;
+
+  if (isMarkdownCodeBlock) {
+    // JSX source blocks are passed without the backticks
+    const language = rawValue.indexOf('```') == 0 ? rawValue.substring(3, rawValue.indexOf('\n')) : 'jsx';
```

Comment:
nit: I don't think we're shooting for IE11 support anymore, might be worthwhile to finally use modern javascript

```suggestion
    const language = rawValue.includes('```') ? rawValue.substring(3, rawValue.indexOf('\n')) : 'jsx';
```

Reply:
Will I do agree, I also think it's best to keep as is until there is a definite decision on the topic.

Reply:
Storybook doesn't work in IE 11, so anything running within storybook also doesn't need to support IE 11.

Reply:
Well then that makes the decision easier. Applying the suggestion!

Reply:
> Storybook doesn't work in IE 11

In general it does AFAIK. there are some glitches regarding Docs (mdx and friends)